src/hevm/src/EVM.hs

```diff
@@ -2693,6 +2751,20 @@ costOfCreate (FeeSchedule {..}) availableGas' hashSize =
     hashCost   = g_sha3word * ceilDiv (num hashSize) 32
     initGas    = allButOne64th (availableGas - createCost)
 
+concreteModexpGasFee :: ByteString -> Integer
+concreteModexpGasFee input = num $ max 200 ((multiplicationComplexity * iterCount) `div` 3)
+  where (lenb, lene, lenm) = parseModexpLength input
+        ez = isZero (96 + lenb) lene input
```

Comment:
where does the `96` come from?

Reply:
input is given on the following form:
`<length_of_BASE> <length_of_EXPONENT> <length_of_MODULUS> <BASE> <EXPONENT> <MODULUS>`
the first three are all 32 bytes. therefore `<BASE>` begins at 96